docs/index.md

```diff
@@ -15,46 +15,48 @@ In order to integrate with Lead Safe, the health provider will need to make a RE
-| lab.type       | Text     | BLL                                                        | Static BLL unless we identify additional labs to include                                                                                                                                                                    |
-| lab.date       | Date     | yyyy-mm-dd                                                 |                                                                                                                                                                                                                             |
-| lab.route      | Text     | V                                                          | *Always "V"  Veinous for Alliance sites                                                                                                                                                                                     |
-| lab.result     | Text     |                                                            | Several values are possible; integer, non-integer numeric, ranges indicated by alphanumeric text.  We'll use previous examples to essentially construct a dictionary.                                                       |
+| Field           | Format   | Constraints                                                | Notes/Questions                                                                                                   |
+|-----------------|----------|------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------|
+| timestamp       | TimeDate | yyyy-mm-dd hh:mm:ss.sss-hh:mm                              | Current Timestamp. RFC 3339 (ISO 8601 derivative).                                                                |
+| visit_date      | Date     | yyyy-mm-dd                                                 | Effective date of score                                                                                           |
+| network_id      | Text     |                                                            | The parent entity of the submitting organization                                                                  |
+| clinic_id       | Text     |                                                            | Two Characted Organization ID (NN- NearNorth  EF- ErieFamily)                                                     |
+| location_id     | Text     |                                                            | Specific Location Abbreviation in Centricity                                                                      |
+| patient_id      | Text     | AlphaNumeric                                               | A id to identify individual patients, should be unique with secondary key.                                        |
```

Comment:
For Alliance the `patient_id` is only unique within the context of a clinic / location. So one (or both) fields are needed to uniquely identify patients. 

The only reason we asked for the "location" was to uniquely identify patients. When we asked for the location we got two fields, `clinic_id` and `location_id`.  

I think I'll change the `patient_id` description to be simply "patient identifier", I think saying more is just confusing.

Reply:
Also, the `location_id` is different than the `location` in the visit array.  The location in the visit array is just an address. As we discussed, I'll leave that alone.

Reply:
I don't know what to do with the `clinic_id` field.  That seems to be very Alliance specific. I don't know how to modify that description to make sense for a broader audience of hospital groups. 

The `location_id` sounds more standard since that's a Centricity field.